chromium/rules.js

```diff
@@ -250,20 +250,20 @@ RuleSets.prototype = {
 
     // Read user prefs
     if (rule_set.name in this.ruleActiveStates) {
-      rule_set.active = (this.ruleActiveStates[rule_set.name] == "true");
+      rule_set.active = this.ruleActiveStates[rule_set.name];
```

Comment:
Does `rule_set.active`  go from being a bool, to a string?

Reply:
No, simply bad code.